newsroom/auth_server/auth.py

```diff
@@ -13,7 +13,7 @@ class JWTAuth(TokenAuth):
     Implements JWT auth logic.
     """
 
-    def check_auth(self, token, allowed_roles, resource, method):
```

Comment:
I think you have to keep those for the authentication to work later when hooked to eve